extensions/guacamole-auth-cas/src/main/resources/translations/en.json

```diff
@@ -0,0 +1,7 @@
+{
+
+    "LOGIN" : {
+        "FIELD_HEADER_TICKET" : ""
```

Comment:
What's the purpose of defining an empty translation string for the field header?